maquetado/trainningApp.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="css/equalizer.css">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body class="container">
+    <div class="top-bar"></div>
```

Comment:
you don't need this here, you can use a border-top in the following section. this is to avoid adding style related code to the page structure. we should try to avoid that if it's possible.